src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -204,6 +204,58 @@ public IActionResult BatchForecastData(IEnumerable<EmissionsForecastBatchDTO> re
         return Ok(result);
     }
 
+    /// <summary>
+    /// Retrieves the actual data that occurred between the time boundaries and calculates the carbon intensity that occurred during that window. 
+    /// </summary>
+    /// <remarks>
+    ///  This endpoint is useful for reporting the measured carbon intensity for a specific time period in a specific location.
+    /// </remarks>
+    /// <param name="location"> the location in </param>
+    /// <param name="jobStartTime"> the time at which the job and thus carbon usage starts </param>
```

Comment:
🐑 I know we keep talking about param namings but "job" seems specific to "Azure Batch Jobs" versus getting the actual carbon intensity for a time period. I.e., most people don't think in terms of "jobs", that seems more specific to this case.